content/id/docs/concepts/containers/images.md

```diff
@@ -0,0 +1,370 @@
+
+Kamu membuat Docker _image_ dan mengunduhnya ke sebuah registri sebelum digunakan di dalam Kubernetes Pod.
+
+Properti `image` dari sebuah Container mendukung sintaksis yang sama seperti perintah `docker`, termasuk registri privat dan _tag_.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Memperbarui Image
+
+Kebijakan _pull default_ adalah `IfNotPresent` yang membuat Kubelet tidak
+lagi mengunduh (_pull_) sebuah image jika sudah ada terlebih dahulu. Jika kamu ingin agar
+selalu diunduh, kamu bisa melakukan salah satu dari berikut:
+
+- mengatur `imagePullPolicy` dari Container menjadi `Always`.
+- buang `imagePullPolicy` dan gunakan `:latest` _tag_ untuk _image_ yang digunakan.
+- buang `imagePullPolicy` dan juga _tag_ untuk _image_.
+- aktifkan [AlwaysPullImages](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages) _admission controller_.
+
+Harap diingat kamu sebaiknya hindari penggunaan _tag_ `:latest`, lihat [panduan konfigurasi](/docs/concepts/configuration/overview/#container-images) untuk informasi lebih lanjut.
+
+## Membuat Image Multi-arsitektur dengan Manifest
+
+Docker CLI saat ini mendukung perintah `docker manifest` dengan anak perintah `create`, `annotate`, dan `push`. Perintah-perintah ini dapat digunakan
+untuk membuat (_build_) dan mengunggah (_push_) _manifest_. Kamu dapat menggunakan perintah `docker manifest inspect` untuk membaca _manifest_.
+
+Lihat dokumentasi docker di sini:
+https://docs.docker.com/edge/engine/reference/commandline/manifest/
+
+Lihat contoh-contoh bagaimana kami menggunakan ini untuk proses _build harness_:
+https://cs.k8s.io/?q=docker%20manifest%20(create%7Cpush%7Cannotate)&i=nope&files=&repos=
+
+Perintah-perintah ini bergantung pada Docker CLI, dan diimplementasi hanya di sisi CLI. Kamu harus mengubah `$HOME/.docker/config.json` dan mengatur _key_ `experimental` untuk mengaktifkan
+atau cukup dengan mengatur `DOCKER_CLI_EXPERIMENTAL` variabel _environment_ menjadi `enabled` ketika memanggil perintah-perintah CLI.
+
+{{< note >}}
+Gunakan Docker *18.06 ke atas*, versi-versi di bawahnya memiliki _bug_ ataupun tidak mendukung perintah eksperimental. Contohnya https://github.com/docker/cli/issues/1135 yang menyebabkan masalah di bawah containerd.
+{{< /note >}}
+
+Kalo kamu terkena masalah ketika mengunggah _manifest-manifest_ yang rusak, cukup bersihkan _manifest-manifest_ yan glama di `$HOME/.docker/manifests` untuk memulai dari bersih.
+
+Untuk Kubernetes, kami biasanya menggunakan _image-image_ dengan sufiks `-$(ARCH)`. Untuk kompatibilitas (_backward compatibility_), lakukan _generate image-image_ yang lama dengan sufiks. Idenya adalah men-_generate_, misalnya `pause` image yang memiliki _manifest_ untuk semua arsitektur dan misalnya `pause-amd64` yang punya kompatibilitas terhadap konfigurasi-konfigurasi lama atau berkas-berkas YAML yang bisa saja punya _image-image_ bersufiks yang di-_hardcode_.
+
+## Menggunakan Registri Privat (_Private Registry_)
```

Comment:
```suggestion
## Menggunakan Registri Privat (_Private Registry_) {#menggunakan-registri-privat}
```

Reply:
thanks @sftim for pointing this out, I think this is a good idea, we can introduce this for other pages as well

Reply:
fixed in https://github.com/kubernetes/website/pull/19784/commits/3fb85d866101c08ce75ea0265beac4929e73f29c